include/roaring/ordered_writer.h

```diff
@@ -0,0 +1,18 @@
+#include <assert.h>
+#include <roaring/roaring.h>
+
+typedef struct roaring_bitmap_writer_s {
+    uint64_t *bitmap;
```

Comment:
This implementation is the exact copy of bitmap_container_t. I think it is better to replace `bitmap` and `cardinality` fields with bitmap_container_t* instance. This will also allow us to make `roaring_bitmap_writer_flush()` more efficient (see other comments).

Reply:
I actually removed `cardinality`, it was there just because I was using the same code as bitset_container_t